test/e2e/app-dir/app-static/app/stale-cache-serving/app-page/page.tsx

```diff
@@ -1,4 +1,4 @@
-export const dynamic = 'force-dynamic'
+export const revalidate = 0
```

Comment:
Updated this so that the existing test that relies on dynamic rendering is preserved. We're able to do this because `revalidate: 0` won't adjust fetch cache behavior if the fetch itself has a revalidate parameter ([docs](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#revalidate))